feeds/wlan-ap/opensync/src/platform/openwrt/src/command/src/task.c

```diff
@@ -106,11 +121,13 @@ void task_status(struct task *task, int status, char *result)
 		return;
 
 	if(!task->handler->doNotQueue) {
+		clear_entry_from_table(task);
```

Comment:
Could move this call just above the "if" statement since it is the first action in both branches.

Reply:
done